src/lib/generated/config.d.ts

```diff
@@ -0,0 +1,63 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type Color = string;
```

Comment:
this was previously a template literal type. The generator doesn't support `format` so it's difficult to encode pattern rules